cluster/gce/gci/configure-helper.sh

```diff
@@ -376,6 +376,10 @@ function mount-ext(){
 # Local ssds, if present, are mounted or symlinked to their appropriate
 # locations
 function ensure-local-ssds() {
+  if [ "${NODE_LOCAL_SSDS_EPHEMERAL:-false}" == "true" ]; then
```

Comment:
Do we need more plumbing of this env? Like in configure-default.sh, configure-test.sh?

Reply:
What are the effects or those?

Reply:
Those are used for kube-up.sh and kubetest

Reply:
Done